wiki/en/Running-a-Server.md

```diff
@@ -50,55 +49,47 @@ Unless you plan on hosting more than about 5 players on a slower-speed home conn
 
 - Running a server may require you to adjust any firewalls running on or outside of your machine or cloud host.
 
-- Running a Private Server at home will require you to [port forward](#running-a-private-server) on your router. When running a Public Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
+- Running an Uregistered Server at home will require you to [port forward](#running-an-unregistered-server) on your router. When running a Registered Server, port forwarding should not be necessary in most cases, but it's advisable to do so because some networks may not work properly with Jamulus in its default mode.
 
 - Jamulus only has limited IPv6 support which needs to be enabled with a command line argument on the client and server. There are plans to expand IPv6 support.
 
-
 # Server Types
 
-You can run your server in one of three "modes" (either at home or on a 3rd party host):
+You can run your server in different ways (either at home or on a 3rd party host):
 
-### 1. Public
+### 1. Registered 
 
-Use this mode when you want anyone to join your server, whoever they are.   
-
-Your server will be listed in the Directory that clients use by default. Musicians can then discover and connect to your server.
+Your server will be listed in a Directory. By default, Jamulus has a list of Public Directories that clients can connect to. If you register with one of these, anyone can then discover and connect to your server. 
 
 <figure>
-	<img src="{% include img/en-screenshots/diagram-public-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Public Server">
-<figcaption>How Public Servers work</figcaption>
+	<img src="{% include img/en-screenshots/diagram-reg-server.inc %}" loading="lazy" alt="Diagram of connections between clients within a Jamulus Registered Server">
+<figcaption>How Registered Servers work</figcaption>
 </figure>
 
+### 2. Unregistered 
 
-
-### 2. Private
-
-This is the default when starting a server for the first time. Private Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not (in any server mode) let you control who can connect to a server.
-
+This is the default when starting a server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful because Jamulus does not let you control who can connect to a server.
```

Comment:
```suggestion
This is the default when starting a server for the first time. Unregistered Servers are not listed by Directories, so only musicians who know your server's address to will be able to connect to it. This is useful if you don't want your server to be public since Jamulus does not let you control who can connect to a server.
```

... or published

Reply:
Again, the use of the term "public" there might be misleading because you might be connecting your server to a private directory (eg on a LAN). This kind of "correctness" is the point of this PR.

Reply:
> if you don't want your server to be public since

I think the simpler version ("because") is clearer and less confusing (without any change).